new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -244,6 +256,34 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
       logger.info("Using image {0}", kindRepoImage);
       image = kindRepoImage;
       isUseSecret = false;
+
+      // We do not know why the kind clusters can't pull Apache webtier image from OCIR using the image pull secret.
```

Comment:
We should not put any open ended question in the  test description, rather says for fin cluster we need to follow these steps.